bigframes/operations/structs.py

```diff
@@ -57,3 +57,26 @@ def dtypes(self) -> pd.Series:
             ],
             index=[pa_type.field(i).name for i in range(pa_type.num_fields)],
         )
+
+
+@log_adapter.class_logger
+class StructFrameAccessor(vendoracessors.StructFrameAccessor):
+    __doc__ = vendoracessors.StructAccessor.__doc__
+
+    def __init__(self, data: bigframes.dataframe.DataFrame) -> None:
+        self._parent = data
+
+    def explode(self, column, separator: str = ".") -> bigframes.dataframe.DataFrame:
+        df = self._parent
+        column_labels = bigframes.core.explode.check_column(column)
+
+        for label in column_labels:
```

Comment:
TODO: Add test with multiple columns to explode.

Reply:
Done in https://github.com/googleapis/python-bigquery-dataframes/pull/916/commits/6d2a84e2a406c5c9203607b99d342fc5298f24a5